examples/03_evaluate/als_movielens_diversity_metrics.ipynb

```diff
@@ -153,7 +153,7 @@
     "import pyspark\n",
```

Comment:
<p>Is there a reason we want to change COL_ITEM? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1538/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Not particular :D. I think this is an artifact of me pasting from the default column constant. I can change it back to "MovieId".</p>

Reply:
Done